daml-lf/transaction/src/main/scala/com/digitalasset/daml/lf/transaction/Transaction.scala

```diff
@@ -384,6 +384,7 @@ object Transaction {
    *        time.
    */
   final case class MetaData(
```

Comment:
nitpick: should be spelled "Metadata" as it's one word.

Reply:
nitpicking are welcome